oat-sa/tao-core-ui-fe
# ACTP-312 Add accessibility compatibility to login errors

https://oat-sa.atlassian.net/browse/ACTP-312

Add accessibility compatibility to login errors

**How to test:**
- checkout /tao-core-ui-fe repo
- switch to branch `feature/ACTP-312/add-accessibility-compatibility-to-login-errors`
- go to repo directory and run `npm link`
- go to your TAO installation directory
- go to path `tao/views` subdir
- run `npm link @oat-sa/tao-core-ui-fe`
- go to path `tao/view/build` subdir
- run `npm install` to install depencies
- run `npx grunt sass:tao` to build styles
- open TAO login url and try to enter wrong data for show the error message
- check shown popup for accessibility compatibility

**Related documents**
https://oat-sa.atlassian.net/wiki/spaces/OP/pages/203096620/Investigate+Pages+in+TestRunner+on+Screen+Reader+Access+and+Navigation+LOG+IN+INDEX+PROCTOR#InvestigatePagesinTestRunneronScreenReaderAccessandNavigation(LOGIN,INDEX,PROCTOR)-Addaccessibilitycompatibilitytologinerrors
